client/src/app/pages/identities/components/identity-form/kind-bearer-token-form.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import { useFormContext } from "react-hook-form";
+
+import { Identity } from "@app/api/models";
+import KeyDisplayToggle from "@app/components/KeyDisplayToggle";
+import { HookFormPFTextInput } from "@app/components/HookFormPFFields";
+
+export const KindBearerTokenForm: React.FC<{ identity?: Identity }> = ({
+  identity,
+}) => {
+  const { control, getValues, resetField } = useFormContext();
+  const values = getValues();
+
+  const [isKeyHidden, setIsKeyHidden] = useState(true);
+  const toggleHideKey = (e: React.FormEvent<HTMLElement>) => {
+    e.preventDefault();
+    e.stopPropagation();
+    setIsKeyHidden(!isKeyHidden);
+  };
+
+  const isKeyEncrypted = identity?.key === values.key;
+
+  return (
+    <HookFormPFTextInput
+      control={control}
+      name="key"
+      label={"Token"}
+      fieldId="key"
```

Comment:
seems we are OK here but `key` is a dangerous name for a prop:)

Reply:
...history ;-)